src/data/ssdp.ts

```diff
@@ -7,6 +7,7 @@ import type { Store } from "home-assistant-js-websocket/dist/store";
 import type { DataTableRowData } from "../components/data-table/ha-data-table";
 
 export interface SSDPDiscoveryData extends DataTableRowData {
+  name: string | undefined;
```

Comment:
It should be very rare that this is undefined, but not all devices are within spec